bsp/bl808/m0/drivers/drv_gpio.c

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (c) 2006-2023, RT-Thread Development Team
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Change Logs:
+ * Date           Author       Notes
+ * 2023/01/5      chushicheng  first version
+ *
+ */
+
+#include "drv_gpio.h"
+#include <stdbool.h>
+#include "bl808_gpio.h"
+#include "bl808_glb.h"
+#include "bl808.h"
+
+#ifdef RT_USING_PIN
+
+#define DBG_TAG              "drv.gpio"
+#define DBG_LVL               DBG_INFO
+#include <rtdbg.h>
+
+static void GPIO0_IRQHandler(void);
+
+struct gpio_int_cfg_private {
```

Comment:
格式还是有问题，我没有表达清楚，这个花括号要放下来
```suggestion
struct gpio_int_cfg_private
{
```

整个文件都是同理，花括号要独占一行